web/src/app/pages/create-survey/task-details/task-input/option-editor/option-editor.component.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Component,
+  OnInit,
+  Input,
+  Output,
+  EventEmitter,
+  OnChanges,
+  ViewChild,
+  ElementRef,
+} from '@angular/core';
+import {
+  FormGroup,
+  FormBuilder,
+  Validators,
+  AbstractControl,
+} from '@angular/forms';
+
+@Component({
+  selector: 'ground-option-editor',
+  templateUrl: './option-editor.component.html',
+  styleUrls: ['./option-editor.component.scss'],
+})
+export class OptionEditorComponent implements OnInit, OnChanges {
```

Comment:
Please name `EditOptionComponent` for consistency with `EditSurvey` and `EditJob`. The only other component with this naming convention is `InlineEditorComponent`, but that will be removed in the latest design.